core/src/test/java/org/elasticsearch/index/engine/InternalEngineTests.java

```diff
@@ -175,6 +176,62 @@
 
 public class InternalEngineTests extends EngineTestCase {
 
+    public void testVersionMapAfterAutoIDDocument() throws IOException {
+        ParsedDocument doc = testParsedDocument("1", null, testDocumentWithTextField(),
```

Comment:
can we add an explicit "test" field value? Using the default value makes it harder to understand what the searcher finds later on.